src/lib/y2storage/storage_manager.rb

```diff
@@ -59,7 +59,25 @@ class StorageManager
     # @return [Proposal, nil]
     attr_reader :proposal
 
-    def_delegators :@storage, :environment, :arch
+    def_delegators :@storage, :environment, :arch, :rootprefix
+
+    # @!method rootprefix
+    #   @return [String] root prefix used by libstorage
+
+    # Sets the root prefix used by libstorage in subsequent operations
+    #
+    # @param path [String]
+    def rootprefix=(path)
+      storage.rootprefix = path
+    end
```

Comment:
I tried and it didn't work. The tests failed. Something to do with SWIG, I assumed.